<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractWindow</class>
 <widget class="QDialog" name="ExtractWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frog Archiver - Extract</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FrogArchiver.qrc">
    <normaloff>:/assets/frog_mac_archigraphs.png</normaloff>:/assets/frog_mac_archigraphs.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input Frog File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(237, 212, 0);</string>
       </property>
       <property name="text">
        <string>Select .frog File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="extractPosition">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputDirButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(237, 212, 0);</string>
       </property>
       <property name="text">
        <string>Select Output Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(78, 154, 6);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(138, 226, 52);</string>
       </property>
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="FrogArchiver.qrc"/>
 </resources>
 <connections/>
</ui>
